code/modules/reagents/chemistry/reagents/misc.dm

```diff
@@ -632,3 +632,50 @@
 	description = "Ewwwwwwwww."
 	reagent_state = LIQUID
 	color = "#857400"
+
+/datum/reagent/spraytan
```

Comment:
This is a huge meme (and so is the whole beach ruin). 

/tg/ allows anyone to make spraytan with I think oil and something else, but I don't think that's suited for Paradise. So this stuff is only available on the beach ruin (if we even want to keep that enabled).